pytorch_lightning/utilities/fetching.py

```diff
@@ -144,23 +144,18 @@ def _store_dataloader_iter_state(
                 dataloader_iter.state.update(iter_name, state)
 
     @property
-    def loaders(self) -> List[DataLoader]:
```

Comment:
These annotations were not correct. CombinedLoader can return anything really and `apply_to_collection` supports it